doc/user/content/guides/dbt.md

```diff
@@ -0,0 +1,276 @@
+
+    <h5>Creating a materialized view</h5>
+
+    This is where Materialize goes beyond dbt's incremental models (and traditional databases), with [materialized views](/sql/create-materialized-view) that **continually update** as the underlying data changes:
+
+    ```sql
+    {{ config(materialized='materializedview') }}
+
+    SELECT symbol,
+           AVG(bid_price) AS avg
+    FROM {{ ref('market_orders') }}
+    GROUP BY symbol
+    ```
+
+    When should you use what? We recommend using `materializedview` models exclusively for your **core business logic** to ensure that you’re not consuming more memory than needed in Materialize. Intermediate or staging views should use the `view` materialization type instead.
+
+1. [Run](https://docs.getdbt.com/reference/commands/run) the dbt models:
+
+    ```
+    dbt run
+    ```
+
+    This command generates **executable SQL code** from any model files under the specified directory and runs it in the target environment. You can find the compiled statements under `/target/run` and `target/compiled`, in the dbt project folder.
+
+1. Using a new terminal window, [connect](/connect/cli/) to Materialize to double check that all the objects were created:
+
+    ```bash
+    psql -U materialize -h localhost -p 6875 materialize
+    ```
+
+    ```sql
+    materialize=> SHOW SOURCES;
+
+           name
+    -------------------
+     market_orders_raw
+
+     materialize=> SHOW VIEWS;
+
+           name
+    -------------------
+     avg_bid
+     market_orders
+    ```
+
+That's it! From here on, Materialize makes sure that your models are **incrementally updated** as new data streams in, and that you get **fresh and correct results** with millisecond latency whenever you query your views.
+
+### Materializations
+
+dbt models are materialized as `views` by default, but can be configured to use a different materialization type through the `materialized` configuration parameter. This parameter can be set directly in the model file, using:
+
+```sql
+{{ config(materialized='materializedview') }}
+```
+
+Because Materialize is optimized for (near) real-time transformations of streaming data and the core of dbt is built around batch, the `dbt-materialize` adapter implements a few custom materialization types:
+
+{{% dbt-materializations %}}
+
+## Document and test a dbt project
+
+dbt can automatically generate [documentation](https://docs.getdbt.com/docs/building-a-dbt-project/documentation) for your project as a shareable website. This brings **data governance** to your streaming pipelines, speeding up life-saving processes like data discovery (_where_ to find _what_ data) and lineage (the path data takes from source(s) to sink(s), as well as the transformations that happen along the way).
+
```

Comment:
This may be a lack of familiarity with dbt terminology, but I think of "lineage" as a path or a concept rather than a process. Would "identifying lineage" work instead?

Reply:
It's not a dbt-specific term! It's a process under the umbrella of "data governance", same as data discovery.